inc/modules/export/class-table.php

```diff
@@ -0,0 +1,451 @@
+	 *
+	 * @return string Text to be placed inside the column <td>
+	 */
+	public function column_file( $item ) {
+		$html = '<div class="export-file">';
+		$html .= $this->getIcon( $item['file'] );
+		$html .= '<div class="export-file-name">' . esc_html( $item['file'] ) . '</div>';
+		$html .= '</div>';
+
+		$delete_url = sprintf( '/admin.php?page=%s&action=%s&ID=%s', $_REQUEST['page'], 'delete', $item['ID'] );
+		$delete_url = get_admin_url( get_current_blog_id(), $delete_url );
+		$delete_url = esc_url( add_query_arg( '_wpnonce', wp_create_nonce( 'bulk-files' ), $delete_url ) );
+		$onclick = 'onclick="if ( !confirm(\'' . esc_attr( __( 'Are you sure you want to delete this?', 'pressbooks' ) ) . '\') ) { return false }"';
+		$actions['delete'] = sprintf(
+			'<a href="%s" aria-label="%s" ' . $onclick . '>%s</a>',
+			$delete_url,
+			/* translators: %s: filename */
+			esc_attr( sprintf( __( 'Delete &#8220;%s&#8221;' ), $item['file'] ) ),
+			_x( 'Delete', 'verb' )
+		);
+
+		$download_url = get_admin_url( get_current_blog_id(), "/admin.php?page=pb_export&download_export_file={$item['file']}" );
+		$actions['download'] = sprintf(
+			'<a href="%s" aria-label="%s" >%s</a>',
+			$download_url,
+			/* translators: %s: filename */
+			esc_attr( sprintf( __( 'Download &#8220;%s&#8221;' ), $item['file'] ) ),
+			_x( 'Download', 'verb' )
+		);
+
+		$html .= $this->row_actions( $actions );
+
+		return $html;
+	}
+
+	/**
+	 * @param array $item A singular item (one full row's worth of data)
+	 *
+	 * @return string Text to be placed inside the column <td>
+	 */
+	public function column_pin( $item ) {
+		$format = $this->getTinyHash( $item['format'] );
+		$html = "<input type='checkbox' name='pin[{$item['ID']}]' value='{$format}' " . checked( $item['pin'], true, false ) . '/>';
+		return $html;
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_columns() {
+		return [
+			'cb' => '<input type="checkbox" />',
+			'file' => __( 'File', 'pressbooks' ),
+			'format' => __( 'Format', 'pressbooks' ),
+			'size' => __( 'Size', 'pressbooks' ),
+			'pin' => '📌', // <span class="dashicons dashicons-admin-post"></span> or 📌
+			'exported' => __( 'Exported', 'pressbooks' ),
+		];
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_sortable_columns() {
+		$sortable_columns = [
+			'file' => [ 'file', true ],
+			'format' => [ 'format', true ],
+			'pin' => [ 'pin', true ],
+			'exported' => [ 'exported', true ],
+		];
+		return $sortable_columns;
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_bulk_actions() {
+		return [
+			'delete' => 'Delete',
+		];
+	}
+
+	/**
+	 * Prepares the list of items for displaying.
+	 */
+	public function prepare_items() {
+
+		// Define Columns
+		$columns = $this->get_columns();
+		$hidden = [];
+		$sortable = $this->get_sortable_columns();
+		$this->_column_headers = [ $columns, $hidden, $sortable ];
+
+		// Data
+		$data = $this->getLatestExports();
+
+		// Pagination
```

Comment:
Not sure this will be an issue but best to check out: https://make.wordpress.org/core/2019/01/14/new-styling-for-admin-table-pagination-links-in-wordpress-5-1/